app/views/Dashboard/Overview/index.tsx

```diff
@@ -27,10 +31,31 @@ function Overview(props: Props) {
     // TODO: define strict type mapMode and tableMode instead of string
     const [currentTab, setCurrentTab] = useState<string | undefined>('mapMode');
 
+    const [
+        mapModalShown,
+        setMapModalVisible,
+        setMapModalHidden,
+    ] = useModalState(false);
+
+    const handleMapModal = () => {
+        console.log('Handle map modal:::>>>>>', mapModalShown);
```

Comment:
remove this. Also, we can directly use setMapModalVisible in the function.
Also, rename setMapModalVisible to showMapModal